src/NServiceBus.Core/Headers.cs

```diff
@@ -223,5 +223,26 @@ public static partial class Headers
         /// of the message can be known.
         /// </summary>
         public const string TimeToBeReceived = "NServiceBus.TimeToBeReceived";
+
+        /// <summary>
+        /// Traceparent header according to the W3C spec:
+        /// https://www.w3.org/TR/trace-context/#traceparent-header
+        /// 23 November 2021. 
+        /// </summary>
+        public const string DiagnosticsTraceParent = "traceparent";
```

Comment:
Could these conflict if the transport natively also implement OTel?

Reply:
yep, absolutely. We're discussing using a dedicated header name to prevent any clashes with transport packages/SDKs that follow the spec.